packages/redux-routine/src/runtime.ts

```diff
@@ -13,18 +16,22 @@ import { isActionOfType, isAction } from './is-action';
 /**
  * Create a co-routine runtime.
  *
- * @param {Object}   controls Object of control handlers.
- * @param {Function} dispatch Unhandled action dispatch.
- *
- * @return {Function} co-routine runtime
+ * @param  controls Object of control handlers.
+ * @param  dispatch Unhandled action dispatch.
  */
-export default function createRuntime( controls = {}, dispatch ) {
+export default function createRuntime(
+	controls: Record<
+		string,
+		( value: any ) => Promise< boolean > | boolean
+	> = {},
+	dispatch: Dispatch
+) {
 	const rungenControls = map(
 		controls,
-		( control, actionType ) => (
+		( control, actionType ): Control => (
 			value,
-			next,
-			iterate,
+			_,
+			__,
```

Comment:
Was this necessary? I see the `tsconfig` turns off `noUnusedParameters`. Only one or the other should be necessary.

If we keep this, it seems better to keep a readable name while flagging them as "ok to not use" with a `_` prefix:

```suggestion
			_next,
			_iterate,
```